pkg/occlient/occlient.go

```diff
@@ -755,6 +761,44 @@ func addBootstrapInitContainer(dc *appsv1.DeploymentConfig, dcName string) {
 		})
 }
 
+// addBootstrapSupervisordInitContainer creates an init container that will copy over
+// supervisord to the application image during the start-up procress.
+func addBootstrapSupervisordInitContainer(dc *appsv1.DeploymentConfig, dcName string) {
+
+	// TODO: Use a release process to use an appriate image based on the release (0.1.0, HEAD, etc.)
+	// TODO: Maybe even make this a configurable container (for development at least too), since this will
+	// be pulled publically
+	image := "docker.io/cdrage/supervisord-test:latest"
```

Comment:
it might be better use quay.io 
I've already created openshiftdo org there. If you send me your username on quay.io I'll give you rights.